lib/chef/provider/registry_key.rb

```diff
@@ -70,6 +70,13 @@ def values_to_hash(values)
         end
       end
 
+      def key_missing?(values, name)
+        values.each do |v|
+          return true unless v.has_key?(name)
+        end
+        return false
```

Comment:
this could be just `false`

https://github.com/bbatsov/ruby-style-guide#no-explicit-return